### DISCUSSION TITLE
Stream from OBS

### DISCUSSION THREAD

**Discussion by fgadaleta** [scottlamb/moonfire-nvr#199] [Q&A]
Is there a way to stream from software like OBS? 
Is there an RTSP server that is launched? Or does moonfire simply connect to an IP camera for recording?

**Comment by scottlamb** [comment]
Moonfire connects to an RTSP stream and records it (and allows live view via its web UI). If OBS can export a RTSP stream, then yes Moonfire NVR should be able to record it, though I've never tried.

**Comment by valpackett** [comment]
You could try the [obs-rtspserver](https://github.com/iamscottxu/obs-rtspserver) plugin, or stream via RTMP to a server like [rtsp-simple-server](https://github.com/aler9/rtsp-simple-server) that would host an RTSP stream for Moonfire to consume.
